packages/amazonq/.vscode/tasks.json

```diff
@@ -13,27 +13,37 @@
                 "kind": "build",
                 "isDefault": true
             },
-            "dependsOn": ["serve"]
+            "dependsOn": ["watchCore", "webpackCore", "serveVueCore"]
         },
         {
```

Comment:
Is this extracted from https://github.com/aws/aws-toolkit-vscode/pull/4739/files or vice versa?

Reply:
Yeah its from there. Our `toolkit` and `amazonq` tasks are gonna be very similar